docs/api_call.md

```diff
@@ -1,1352 +1,1108 @@
 
-      ```python
-      from ner_v1.chatbot.entity_detection import get_phone_number
-      output = get_phone_number(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
+4. **fallback_value**: It is a fallback value. If the detection logic fails to detect any value either from *structured_value* or *message* then we return a *fallback_value* as an output. This value is derived from third party api or from users profile.
 
-    - *CURL command:*
+   For example, if user says *"Nearby ATMs"*. In this example user has not provided any information about his location in the chat but, we can pass a *fallback_value* that will contain its location that can be obtained from its profile or third party apis (like geopy, etc).
 
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
+5. **source_language**: It is the language in which user message is passed. We have to pass ISO 639-1 code of the language here.
 
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/phone_number/?message=my%20contact%20number%20is%209049961794&entity_name=phone_number&structured_value=&fallback_value=None&bot_message='
-      ```
+6. **bot_message**: previous message from a bot/agent. This is an important parameter, many times the entity value relies on the message from the bot/agent i.e. what is bot saying? or asking?
 
-    - *CURL Output:*
+   For example, bot might ask for departure date to book a flight and user might reply with a date. Now, it is difficult to disambiguate whether its departure date or arrival date unless, we know what bot is asking for?
 
-      ```json
-      {
-        "data": [
-          {
-            "detection": "message",
-            "original_text": "9049961794",
-            "entity_value": {
-              "value": "9049961794"
-            }
-          }
-        ]
-      }
-      ```
+   ```
+   bot: Please help me with date of departure?
+   user: 23rd March
+   ```
 
-  - Example 2: 
+### **Output Format**
 
-    - ```python
-      message = 'Please call me'
-      entity_name = 'phone_number'
-      structured_value = None
-      fallback_value = '9049961794'
-      bot_message = None
-      ```
+The output of entities detected will be stored in a list of dictionary containing the following structure:
 
-    - *Python:* 
+```
+[
+    {
+        "entity_value": entity_value,
+        "detection": detection_method,
+        "original_text": original_text
+    },
+]
+```
 
-      ```python
-      from ner_v1.chatbot.entity_detection import get_phone_number
-      output = get_phone_number(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
+Consider the following example for detailed explanation:
 
-    - *CURL command:*
+```
+"I want to order from mcd"
```

Comment:
Please use number detection example. Text entities are strings and hence is not the right example to use here. Also dish and cuisine are just too outdated.